__tests__/server.test.js

```diff
@@ -93,4 +92,48 @@ describe('GET', () => {
         });
     });
 
-});
+    describe('/api/articles', () => {
+        test('200 - an articles array of article objects with the correct properties', () => {
+            
+            return request(app)
+            .get('/api/articles')
+            .expect(200)
+            .then(({body}) => {
+                body.forEach((article) => {  
+                    expect(article).toHaveProperty('author', expect.any(String))
+                    expect(article).toHaveProperty('title', expect.any(String))
+                    expect(article).toHaveProperty('article_id', expect.any(Number))
+                    expect(article).toHaveProperty('topic', expect.any(String))
+                    expect(article).toHaveProperty('created_at', expect.any(String))
+                    expect(article).toHaveProperty('votes', expect.any(Number))
+                    expect(article).toHaveProperty('article_img_url', expect.any(String))
+                    expect(article).toHaveProperty('comment_count', expect.any(Number))
+                    expect(article).not.toHaveProperty('body')
+                })
+            })
+        });
+        
+        test('200 - articles should be sorted by date descending', () => {
+            
+            return request(app)
+            .get('/api/articles')
+            .expect(200)
+            .then(({body}) => {
+                expect(body).toBeSortedBy('created_at', {
+                    descending: true
+                })
+            })
+        });
+
+        test('404 - receive a 404 when url entered is misspelt', () => {
+            
```

Comment:
this test is fine however shouldn't be in this describe block as more of a generic test. I would also check for an error message